llvm/lib/Analysis/ScalarEvolution.cpp

```diff
@@ -3547,6 +3547,25 @@ const SCEV *ScalarEvolution::getUDivExpr(const SCEV *LHS,
     }
   }
 
+  // ((-C + (C smax %x)) /u %x) evaluates to zero, for any positive %x and any
+  // positive constant C.
```

Comment:
```suggestion
  // ((-C + (C smax %x)) /u %x) evaluates to zero, for any
  // positive constant C.
```
Positive %x not required?